custom_components/tuya_local/devices/stadlerform_roger_purifier.yaml

```diff
@@ -25,7 +25,16 @@ primary_entity:
         - dps_val: auto
           value: 100
 secondary_entities:
+  - entity: switch
+    name: Child lock
+    class: switch
+    icon: "mdi:lock-outline"
+    dps:
+      - id: 7
+        name: switch
+        type: boolean
   - entity: light
+    name: Light
```

Comment:
Entities should not have the same name as the entity type, as it shadows default translations.